Informasi-Jadwal-Praktikum.md

```diff
@@ -0,0 +1,36 @@
+## PRA TUBES CALON ASISTEN (CA) 2024 ICLABS FIKOM UMI (WEB)
+
+## NAMA APLIKASI :smiling_imp:
+> [Informasi-Jadwal-Praktikum]
+> MUHAMMAD ALIF MAULANA.R - 13020220223
+
+
+> [!IMPORTANT]
+> Key information users need to know to achieve their goal.
+
+## THE PATH
+...
+> PREPARE
+>> Install Discord
+>> Add Plugin PlantUML
```

Comment:
Done to add plugin plantUML in VSC

Reply:
fix it doesn't still not match the template